src/main/java/fi/otavanopisto/restfulptv/server/organizations/OrganizationEntityEvictor.java

```diff
@@ -96,7 +96,7 @@ public void timeout(Timer timer) {
   private void checkOrganization(String organizationId) {
     ApiResponse<VmOpenApiOrganization> response = ptvApi.getOrganizationApi().apiOrganizationByIdGet(organizationId);
     if (response.getStatus() == 404) {
-      evictOrganization(organizationId);
+      //evictOrganization(organizationId);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)